easybuild/easyconfigs/s/SLICOT/slicot-makefile-flexiblas.patch

```diff
@@ -0,0 +1,29 @@
+Patch to make the make.inc file EasyBuild compatible
+Author: J. Sassmannshausen (Imperial College London/UK)
+diff --git a/slicot-5.0+20101122.orig/make.inc b/slicot-5.0+20101122/make.inc
+index 3136485..c1458e9 100755
+--- a/slicot-5.0+20101122.orig/make.inc
++++ b/slicot-5.0+20101122/make.inc
+@@ -16,10 +16,10 @@
+ #  selected.  Define LOADER and LOADOPTS to refer to the loader and 
+ #  desired load options for your machine.
+ #
+-FORTRAN  = f77
+-OPTS     = -O4 -native -u
+-NOOPT    = -u
+-LOADER   = f77
++FORTRAN  = gfortran
++OPTS     = -O2 -mtune=native -fPIC -fdefault-integer-8
```

Comment:
@sassy-crick Rather than hardcoding things here, why not use `${FC}` and `${FCFLAGS}` which are set by EasyBuild in the build environment?

Reply:
Good idea. I changed that but I left the -fdefault-integer-8 as that seems to be important. I hope that is not biting what `OpenBLAS` and `FlexiBLAS` are currently having set. Maybe keeping this PR open until we got `-fdefault-integer-8` sorted for them as well?